examples/resolvers-kv/grafbase/grafbase.config.ts

```diff
@@ -13,6 +13,7 @@ const authProvider = auth.Authorizer({
 // https://grafbase.com/docs/database
 
 g.model('Question', {
+    id: g.id(),
```

Comment:
In the Grafbase database, the `id` is implicit and shouldn't be defined in the model.

We're sunsetting the database though so we can soon delete the whole file.

Reply:
:+1: